authentication/templates/email/custom_activation.html

```diff
@@ -0,0 +1,29 @@
+{% load i18n %}
+
+{% block subject %}
+{% blocktrans %}Account activation on {{ site_name }}{% endblocktrans %}
+{% endblock subject %}
+
+{% block text_body %}
+{% blocktrans %}You're receiving this email because you need to finish activation process on {{ site_name }}.{% endblocktrans %}
```

Comment:
Ok, if we are here, can you translate texts in emails to ukrainian?

Reply:
Good point, i`m going to discuss it with BA and rest of the team.